index-handlers/src/test/java/no/sikt/nva/nvi/index/apigateway/FetchInstitutionReportHandlerTest.java

```diff
@@ -314,15 +335,14 @@ void shouldNotContainDataForOtherInstitutions() throws IOException {
   @Test
   void shouldLogCandidateIdOnFailure() throws IOException {
     var topLevelCristinOrg = randomCristinOrgUri();
-    var indexDocument =
-        indexDocumentMissingCreatorAffiliationPoints(CURRENT_YEAR, topLevelCristinOrg);
+    var indexDocument = indexDocumentMissingApprovals(CURRENT_YEAR, topLevelCristinOrg);
     when(openSearchClient.search(any()))
         .thenReturn(aggregationResponse(1))
         .thenReturn(createSearchResponse(indexDocument));
     var appender = LogUtils.getTestingAppender(NviCandidateIndexDocument.class);
     handler.handleRequest(
         requestWithMediaType(MICROSOFT_EXCEL.toString(), topLevelCristinOrg), output, CONTEXT);
-    assertTrue(appender.getMessages().contains(indexDocument.id().toString()));
+    assertTrue(appender.getMessages().contains(indexDocument.identifier().toString()));
   }
```

Comment:
Note that this test used to trigger a different error that did not match the test name and can no longer be easily triggered. This fixes this test case to match the name, but the original exception in `no.sikt.nva.nvi.index.model.document.NviCandidateIndexDocument#generateRow` is now untested.